pkg/server/etcd.go

```diff
@@ -0,0 +1,102 @@
+package server
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/rancher/k3s/pkg/etcd"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func setETCDLabelsAndAnnotations(ctx context.Context, config *Config) error {
```

Comment:
Can we change this function name and/or make it clear via doc comment that it also deletes the control-plane role labels?